app/controllers/volunteers_controller.rb

```diff
@@ -6,6 +6,7 @@ def create
     @volunteer.city = nil if @volunteer.city.blank?
 
     if @volunteer.save
+      VolunteerNotifierMailer.send_registration_email(@volunteer).deliver_now
```

Comment:
Can we use `deliver_later` to keep it out of band?

Reply:
Of course! I switched to `deliver_now` whilst testing, but `delivery_later` makes more sense